css/index.css

```diff
@@ -3,144 +3,355 @@
+    }
+
+
+    .middle-content .boxes {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-evenly;
+    }
+    
+    .middle-content .boxes .box {
+      width: 12.5%;
+      height: 100px;
+      background: black;
+      margin: 20px 2.5%;
+      color: white;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+    
+
+
+.bottom-content {
+  display: flex;
+  margin: 0 2% 20px;
+  justify-content: space-around;
 }
 
-.middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+.bottom-content .text-container {
+  padding-right: 4%;
 }
 
-.middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+.bottom-content .text-container:last-child {
+  padding-right: 0;
 }
 
-.bottom-content {
+
+
+/* Box Section of Image */
+#box1 {
+  background-color: teal;
+}
+
+#box2 {
+  background-color: gold;
+}
+
+#box3 {
+  background-color: cadetblue;
+}
+
+#box4 {
+  background-color: coral;
+}
+
+#box5 {
+  background-color: crimson;
+}
+
+#box6 {
+  background-color: forestgreen;
+}
+
+#box7 {
+  background-color: darkorchid;
+}
+
+#box8 {
+  background-color: hotpink;
+}
+
+#box9 {
+  background-color: indigo;
+}
+
+#box10 {
+  background-color: dodgerblue;
+}
+
+
+.about-top-content{
+ margin-top: 20px;
+ padding-bottom: 20px;
+ border-bottom: 1px dotted black;
+}
+
+
+.about-boxes{
+  display:flex;
+  flex-direction: column;
+}
+  .about-boxes-row{
     display: flex;
-    margin: 0 2% 20px;
-    justify-content: space-around;
+    flex-direction: row;
+    
+  }
+
+.about-boxes-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin: 20px;
+  align-items: flex-start;
+  border-bottom: 1px dashed black;
+
 }
 
-.bottom-content .text-container {
-    padding-right: 4%;
+.header   {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 30px;
+  margin-bottom: 30px;
+} 
+
+.learn-more-button{
+ background-color: white;
+ border: 2px solid black;
+ border-radius: 8px:
+ padding: 5px 10px 5px 10;x
 }
 
-.bottom-content .text-container:last-child {
-    padding-right: 0;
+
+.work-together{
+  margin-top: 20px;
+  margin-bottom: 20px;
+
+}
+
+.about-boxes-item img{
+  margin-bottom: 10px;
 }
 
 footer {
-    width: 100%;
-    background: black;
+  width: 100%;
+  background: black;
 }
 
 footer nav {
-    width: 60%;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+  width: 60%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 2%;
+  font-size: 14px;
 }
 
 footer nav a {
-    color: white;
-    text-decoration: none;
-}
\ No newline at end of file
+  color: white;
+  text-decoration: none;
+}
```

Comment:
Awesome work on the CSS. I love how you are using percents for responsive design!